src/AutoRest.CSharp/Common/Input/InputTypes/Transformation/SubscriptionIdTransformer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+
+namespace AutoRest.CSharp.Common.Input.InputTypes.Transformation
+{
+    internal class SubscriptionIdTransformer
+    {
+        public static void Transform(InputNamespace inputNamespace)
+        {
+            foreach (var client in inputNamespace.Clients)
+            {
+                foreach (var operation in client.Operations)
+                {
+                    var subscriptionIdParameter = operation.Parameters.Where(p => p.NameInRequest == "subscriptionId").FirstOrDefault();
```

Comment:
this could be simplified:
```suggestion
                    var subscriptionIdParameter = operation.Parameters.FirstOrDefault(p => p.NameInRequest == "subscriptionId");
```